packs/hpe-icsp/actions/icsp_buildplan_apply.yaml

```diff
@@ -0,0 +1,28 @@
+---
+  name: "icsp_buildplan_apply"
+  runner_type: "run-python"
+  description: "Apply Buidlplan to Specified Servers"
+  enabled: true
+  entry_point: "icsp_buildplan_apply.py"
+  parameters:
+    buildplan_uris:
+      type: "array"
+      description: "Comma seperated list of build plan URIs"
```

Comment:
Hm, the parameter says URIs, but looking at the code it seems like user just needs to pass a list of integers.

If that's the case, you can also use the `items` jsonschema attribute (http://spacetelescope.github.io/understanding-json-schema/reference/array.html) and it would also be good to update the parameter name (`buildplan_ids` or similar) since URIs usually also contain scheme and other information.